core/mono-samples/iOS/NuGet.config

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <config>
+    <add key="repositoryPath" value="/Users/mdhwang/core/mono-samples/iOS/" />
+  </config>
+  <packageSources>
+    <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
+    <add key="mitchnuget" value="/Users/mdhwang/runtime_wasm_clean/artifacts/packages/Debug/NonShipping/" />
```

Comment:
This was used to test the nuget package locally. Will be modified/removed when this goes in https://github.com/dotnet/runtime/pull/43048